rackcity/models/fields/rackcity_model_fields.py

```diff
@@ -0,0 +1,8 @@
+from django.db.models import PositiveIntegerField
+
+
+class RCPositiveIntegerField(PositiveIntegerField):
+    def get_prep_value(self, value):
+        if value == '':
+            return None
```

Comment:
This is the big change on the model serializer.  Before saving to the database, if the data is "", this will force it to None.  This is relevant when we didn't validate the serializer (like on modify) or (I believe) if we validated the serializer but used the un-validated data anyway

Reply:
cool